charts/jackett/values.yaml

```diff
@@ -17,6 +17,8 @@ env: {}
   # TZ: UTC
   # PUID: 1001
   # PGID: 1001
+  ## Allow Jackett to update inside of the container, but might cause outages if turned on.
+  # AUTOUPDATE: false
```

Comment:
```suggestion
```
This is not really the way containers should run software updates. I don't think we should explicitly mention or support this.

Reply:
This environment variable isn't doing anything anyways, I don't see it in the source of Jackett.

Reply:
It's part of LSIO image, which defaults to `AUTOUPDATE=true`... very odd

Reply:
Yes, the default true was the only reason I added it in.